CHANGELOG.md

```diff
@@ -11,7 +11,11 @@ project adheres to [Semantic Versioning](http://semver.org/).
  - Write your awesome addition here (by @you)
 
 ### Changed
- - Write your awesome change here (by @you)
+
+Fixes:
+- `kubectl apply <aws auth config ma>` is actually tried up to 5 times
+- `terraform apply` fails if invocation of kubectl fails after 5 attempts
```

Comment:
Can you fix this so it's like the other items in the change log? e.g. `Fix zsh-specific syntax in retry loop - @marcelloromani` etc

Reply:
Hi, I've pushed an additional commit to fix this. Please let me know :)

Reply:
Apologies, I've messed up after rebasing against master...